getting-started/setup-building.rst

```diff
@@ -81,13 +81,19 @@ You will only need to execute these steps once:
       $ cd cpython
       $ git remote add upstream git@github.com:python/cpython.git
 
-7. Verify that your setup is correct::
+7. Configure ``git`` to pull main from the ``upstream`` remote::
+
+      $ git config --local branch.main.remote upstream
```

Comment:
So by doing this `git pull` without extra args will pull from `upstream` instead of `origin`?
```suggestion
7. Configure ``git`` to pull ``main`` from the ``upstream`` remote::

      $ git config --local branch.main.remote upstream
```

Reply:
Correct. This configuration is also what happens when one runs `gh repo fork --remote`.